services/analyzer/mysql-table-constraints-getter.js

```diff
@@ -0,0 +1,54 @@
+function MysqlTableConstraintsGetter(databaseConnection) {
+  const queryInterface = databaseConnection.getQueryInterface();
+
+  this.convertToUniqueIndexArray = (constraints) => {
+    const uniqueIndexes = {};
+    constraints.filter((constraint) => constraint.columnType === 'UNIQUE')
+      .forEach((item) => {
+        uniqueIndexes[item.constraintName] = uniqueIndexes[item.constraintName] || [];
+        uniqueIndexes[item.constraintName].push(item.columnName);
+      });
+    const uniqueIndexArray = Object.values(uniqueIndexes);
+    return uniqueIndexArray.length ? uniqueIndexArray : null;
+  };
+
+  this.applyUniqueIndexArray = (constraints) => {
```

Comment:
Again, only a little suggestion
```suggestion
  this.addUniqueConstraintArrayToConstraints = (constraints) => {
```

Reply:
I prefer using 4 words too, sorry :)